group_vars/sn05.yml

```diff
@@ -55,8 +55,10 @@ postgresql_conf:
   - log_min_duration_statement: 100
 postgresql_pg_hba_conf:
   - "host    postgres        galaxy          132.230.223.239/32      md5"
+  - "host    postgres        galaxy-readonly 10.5.68.237/24          md5"
```

Comment:
why is this entry twice in the list?

Reply:
There are 2 entries for sn07 (10.5.68.237/24). 1 is for the postgres database and the 2nd is for the galaxy database where both databases will be accessed by the galaxy-readonly user. Since sn06 has access to both databases I thought sn07 might also need the same. Or did I understand this wrong?